Explorer/Assets/DCL/ExplorePanel/ExplorePanelView.cs

```diff
@@ -0,0 +1,40 @@
+using Cysharp.Threading.Tasks;
+using DCL.UI;
+using DG.Tweening;
+using MVC;
+using System.Threading;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace DCL.ExplorePanel
+{
+    public class ExplorePanelView : ViewBase, IView
+    {
+        [field: SerializeField]
+        public ExploreSections[] Sections { get; private set; }
+
+        [field: SerializeField]
+        public GameObject[] SectionsObjects { get; private set; }
+
+        [field: SerializeField]
+        public RectTransform AnimationTransform { get; private set; }
+
+        [field: SerializeField]
+        public TabSelectorView[] TabSelectorViews { get; private set; }
+
+        [field: SerializeField]
+        public Button CloseButton { get; private set; }
+
+        protected override UniTask PlayShowAnimation(CancellationToken ct)
+        {
+            AnimationTransform.anchoredPosition = new Vector2(0, 1920);

```

Comment:
I'd propose to get these values from the canvas. it is already serialized in the base class

Reply:
We have the root element with the canvas, the problem is that because we use canvas scaler to have UI consistent between different resolutions we need a separate container to animate all the objects to avoid conflicts between canvas scaler and animations with DOTween. Hence I'd like to always have a separate container serialized.

For more info, the canvas scaler affects directly the scale of the RectTransform it's attached to, and I've experienced situations in which it affected the anchoredPosition as well